packages/neptune-extension/src/common/components/ProjectInput.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+
+import { backendClient } from 'common/api/backend-client';
+import { ProjectWithRoleDTO } from 'generated/backend-client/src/models';
+
+interface ProjectInputProps {
+  value: string
+  disabled: boolean
+  onChange: (projectId: string) => void
+}
+
+const ProjectInput: React.FC<ProjectInputProps> = ({
+  value,
+  disabled,
+  onChange: onChangeProp,
+}) => {
+
+  const [ loading, setLoading ] = React.useState(false);
+
+  const [ projectList, setProjectList ] = React.useState<Array<ProjectWithRoleDTO>>(() => {
+    setLoading(true);
+
+    backendClient.api.listProjects({})
```

Comment:
You should use `listProjectsForMemberOrHigher` because it will return only projects that user has write access in.